integration_test/test_xgboost_experiment.py

```diff
@@ -104,6 +104,20 @@ def test_experiment(self, task):
     self._verify_metrics_for_all_runs(sigopt_suggested_runs, experiment.metrics)
     experiment.archive()
 
+  def test_early_stopping(self):
+    task = 'binary'
+    self._form_random_experiment_config(task)
+    self.experiment_params['early_stopping_rounds'] = 1
+    experiment = sigopt.xgboost.experiment(**self.experiment_params)
+    assert experiment.is_finished()
+    sigopt_suggested_runs = list(experiment.get_runs())
+    for ssr in sigopt_suggested_runs:
+      assert ssr.assignments['early_stopping_rounds']
+      for param in experiment.parameters:
+        if param.name == 'num_boost_round':
+          assert ssr.values['num_boost_round_before_stopping']
+          assert ssr.assignments['num_boost_round'] >= ssr.assignments['num_boost_round_before_stopping']
```

Comment:
Based on the assertion, we can not make sure whether it is actually early stopped or not.

Reply:
Yea, I don't think there is any way to guarantee the model early stopped.